book/docs/setup/editors/vscode.md

```diff
@@ -17,4 +18,60 @@ _This guide assumes that you've completed the steps described in [Prerequisites]
    ```
    _Note: if you're using `nix-shell`, make sure to run it first._
 
-If everything is fine you should get auto-completion and other features working 🎉
\ No newline at end of file
+If everything is fine you should get auto-completion and other features working 🎉
+
+---
+</br>
+
+### Additional Troubleshooting Steps
+_In the event the above steps result in the Visual Studio Code Editor not displaying definitions._
+
+### Tested with
+```
+Ubuntu                                          v20.x
+Windows                                         v10.x
+Nix                                             v2.3.10
+Visual Studio Code                              v1.55.2
+VSCode Extension Haskell                        v1.2.0
+VSCode Extension Haskell Syntax Highlighting    v3.4.0
+```
+
+### How to Check
+_Select a function in the Visual Studio Code Editor and right click and choose **Go to Definition** or highlight a function and press **F12**_.
```

Comment:
```suggestion
_Select a function in the Visual Studio Code Editor and right-click and choose **Go to Definition** or highlight a function and press **F12**_.
```

Reply:
Good catch. Resolved in commit c1a467a on `getHashSet/docs.plutus-community.com` branch `editors`.